### DISCUSSION TITLE
new url doesnt work at all

### DISCUSSION THREAD

**Discussion by overtimepog** [jaykali/maskphish#38] [Q&A] [answered]

**Comment by jaykali** [comment]
Can you please provide screenshots then we can help you.

**Comment by jaykali** [comment, marked as answer]
Marked as answered due to inactive

**Reply by ZekePolarisBSH** [reply]
I got the same issue but I can at least say what has happened. The new link far as I know of, isn't getting sent through any platform, even through text message. As far as for this user, I don't know what is wrong.

**Reply by jaykali** [reply]
Please provide us screenshots

**Comment by ZekePolarisBSH** [comment]
How to share a screenshot of a message that never came? I mean Idk how a
screenshot of this can help with fixing this. If anything my phone service
could be blocking it from going? If it helps, I am using a phone from
Metro. I know that all phone services uses a different texting method. Can
be used for anonymous texting using Yahoo.

**Comment by jaykali** [comment]
Ok ok . But maskphish didn't do anything with the URL. it just shorten the URL and add a prefix which is ignored by the browser. We don't understand how messages are not going.

**Comment by ZekePolarisBSH** [comment]
I know what it does that is why I used it lol but yea some reason I guess
it gets blocked by them....

**Reply by jaykali** [reply]
In that case we suggest you to arrange your URL in following method

https://anything.com-social-engineering-keyword@ngrok.io

The browser will ignore all the things before @ and will continue with after @ URL. You can use here ngrok or other ssh port forwarding methods. 

I guess it's blocked our is.gd services.

**Comment by ZekePolarisBSH** [comment]
It doesn't end it in ngrok.io, it ends with random. Maybe that is why
